package.json

```diff
@@ -26,6 +26,7 @@
   ],
   "author": "scorelab",
   "license": "Apache-2.0",
+  "format": "prettier --write ."
```

Comment:
Hey @lumenCodes, the executing scripts will be added to the `scripts` section of the `package.json` file.
```
"scripts": {
    "test": "lerna run test",
    "lint": "lerna run lint",
    "prettier": "lerna run prettier",
    "doc": "jsdoc -c jsdoc.json"
  },
```